src/spectral/utils/spectral-validator.js

```diff
@@ -0,0 +1,44 @@
+const MessageCarrier = require('../../plugins/utils/messageCarrier');
+
+const parseResults = function(results) {
+  const messages = new MessageCarrier();
+
+  if (results) {
+    for (const validationResult of results) {
+      if (validationResult) {
+        const code = validationResult['code'];
+        const severity = validationResult['severity'];
+        const message = validationResult['message'];
+        const path = validationResult['path'];
+
+        if (code && severity != null && message && path) {
+          if (code == 'parser') {
+            // Spectral doesn't allow disabling parser rules, so don't include them
+            // in the output for now
+            continue;
+          }
+          // Our validator only supports warning/error level, so only include
+          // those validation results (for now)
+          if (severity == '1') {
```

Comment:
I thought you said `severity` would be an integer? If it's a string, you can treat it the same in the if statement above

Reply:
It is, but since I'm using loose equality here (``==``) node matched the type. I'll change these conditions to strict equality.